### DISCUSSION TITLE
IIIF Media

### DISCUSSION THREAD

**Discussion by ebraker** [ArctosDB/arctos#4343] [Ideas]
International Image Interoperability Framework (IIIF, "triple I F") is heavily used in Europe and in many cultural collections. It has open source viewing platforms with cool functionality like zooming, [multi-page viewers](https://showcase.iiif.io/showcase/uv/), annotation, [side-by-side comparison](https://projectmirador.org/), a cool historic map media [georeferencing tool ](https://showcase.iiif.io/showcase/georeferencer/) that integrates with google maps, and more. I don't know how or if this would play with TACC media but I wanted to drop it here. Would be a great way to interact with Arctos media.

**What (from website):** IIIF is a set of open standards that help archives, libraries, and museums make the most of their digitized collections with deep zoom, annotation capabilities, and more...The IIIF standards are a set of shared application programming interface (API) specifications for interoperable functionality in digital asset repositories. Using JSON-LD, linked data, and standard W3C web protocols such as Web Annotation, IIIF makes it easy to parse and share digitized materials, migrate across technology systems, and provide enhanced image access for scholars and researchers.

In short, IIIF enables better, faster and cheaper image delivery. It lets you leverage interoperability and the fabric of the Web to access new possibilities and new users for your image-based resources, while reducing long term maintenance and technological lock in. IIIF gives users a rich set of baseline functionality for viewing, zooming, and assembling the best mix of resources and tools to view, compare, manipulate and work with images on the Web, an experience made portable–shareable, citable, and embeddable.

Example - Royal Botanical Garden Edinburgh: https://iiif.rbge.org.uk/

Specs & FAQs: https://iiif.io/community/faq/#what-are-the-benefits-of-iiif

**Comment by dustymc** [comment]
My impression is that IIIF is used by a lot more than cultural collections, eg it seems prevalent in the CT scan community.

I *know* nothing, but I think it's heavily API-centric. Arctos has APIs (and an environment in which they're easily developed) and most Media are stored in places which are accessible to APIs - I don't see any major technical challenges to about any flavor of integration. I believe Harvard uses IIIF as a DAMS and then hooks their Arctos-based system into that via Media, or IIIF could (probably) essentially act as a fancy Arctos media viewer, or ....

**Comment by ebraker** [comment]
> IIIF could (probably) essentially act as a fancy Arctos media viewer

Yes please! Maybe we can add to the next AWG agenda?

**Comment by AJLinn** [comment]
Sounds cool! I'm definitely going to check this out some more! Thanks for bringing it up.

**Comment by Jegelewicz** [comment]
See https://github.com/ArctosDB/arctos/issues/1785#issuecomment-439532288

**Comment by mkoo** [comment]
FYI implementing IIIF for media (digitized field note page and photos) are
part of a submitted grant. So if awarded we should get some bandwidth to
implement and create use cases for media from ARctos to be imported into
digital exhibits proposed to be created on OMEKA

so fingers crossed!
(this is my 2nd attempt to implement IIIF is a workable example for
Arctos-- and probably wont be my last!)

**Comment by ebraker** [comment]
Not sure what viewer MCZbase is using...but the media Zoom feature and general page setup is really nice, e.g.:

https://mczbase.mcz.harvard.edu/MediaSet.cfm?media_id=1504908

**Comment by jldunnum** [comment]
Agreed that would be an excellent addition!

______________________________________________________________

Jonathan L. Dunnum Ph.D.
Senior Collection Manager
Division of Mammals, Museum of Southwestern Biology
University of New Mexico
Albuquerque, NM 87131
(505) 277-9262
Fax (505) 277-1351

MSB Mammals website: http://www.msb.unm.edu/mammals/index.html
Facebook: http://www.facebook.com/MSBDivisionofMammals

Shipping Address:
Museum of Southwestern Biology
Division of Mammals
University of New Mexico
CERIA Bldg 83, Room 204
Albuquerque, NM 87131
________________________________

**Comment by campmlc** [comment]
So what do we need to move forward with this?

**Comment by dustymc** [comment]
A systems engineer would be handy....

I suspect IIIF needs a dedicated VM (or cluster) with a fair bit of CPU and a decent amount of disk for cache, but I can't quantify that.

I'm fairly confident in saying that we don't want IIIF competing with "core" Arctos for computing resources.

There's some "for scale..." information in https://wiki.harvard.edu/confluence/display/LibraryStaffDoc/IIIF+Redesign+Project+Charter - looks like Harvard approached this with about a dozen people, a contract with a consulting firm, and access to their library's impressive computing infrastructure.

**Comment by Jegelewicz** [comment]
Harvard....

**Comment by campmlc** [comment]
From John K: 
To give more context to my request, I am trying to get 3d files into the appropriate format and hosted in such a way that they can be used with the new "Exhibit" online viewer (https://exhibit.so/).

**Comment by Jegelewicz** [comment]
Poster from SPNHC
[IIIF_Poster.pdf](https://github.com/ArctosDB/arctos/files/8923322/IIIF_Poster.pdf)

**Comment by ewommack** [comment]
> so fingers crossed!
> (this is my 2nd attempt to implement IIIF is a workable example for
> Arctos-- and probably wont be my last!)

@mkoo - any update on the grant you can share?

**Comment by dustymc** [comment]
Potential usage: media detail pages are being [rebuilt](https://github.com/ArctosDB/arctos-dev/issues/1), there's plenty of room for something bigger than the thumbnail but still size (pixels and bytes) constrained. I believe that IIIF is very good at making such things.

**Comment by mkoo** [comment]
I've been looking into this more too-- love IIIF enabled API's!

Also check out these embeddable IIIF viewers:
https://iiif.io/get-started/iiif-viewers/
Mirador -  lots of cool options including annotations
Universal Viewer - basic, robust
